sonar-apex-plugin/src/main/java/org/fundacionjala/enforce/sonarqube/apex/ApexSquidSensor.java

```diff
@@ -122,10 +133,7 @@ public ApexSquidSensor(FileSystem fileSystem, ResourcePerspectives perspectives,
      */
     @Override
     public boolean shouldExecuteOnProject(Project project) {
```

Comment:
Improve the name from this method (Example: public boolean executeProject(Project proejct))